SDK.markdown

```diff
@@ -177,6 +177,83 @@ client.auth.cancelMFAConfirmation();
 
 The Promise returned by `getMFAConfirmedToken` will then reject with a `MFAConfirmationCanceledError`.
 
+
+### Device Binding
+To have access to Kontist API endpoints that require strong customer authentication, you need to pass multi-factor authentication (MFA).
+
+To make it seamless for you we provide device binding authentication that leverages digital signature algorithm.
+
+To pass device binding authentication you need to generate a pair of private and public keys by using elliptic curve algorithm (secp256r1). To create and verify your device you need to pass your public key and then sign OTP from SMS received on your phone.
+
+After that, you can use your device id and your private key to get your confirmed auth token by creating and verifying device challenge.
+
+#### Create device
+To initiate the device binding you need to create a device by providing your **public key**.
+
+Please, make sure that you **never share your private key** and it's **stored in a secure place** (do not store it in a browser).
+
+After a successful request, you will receive an SMS with OTP that you will need during device verification.
+
+```typescript
+const result = await clientRef.current.auth.createDevice({
```

Comment:
isn't `clientRef.current` a bit too react oriented for the code example? Maybe `client.auth.createDevice` would be a bit more generic and consistent with other examples in the Authentication section?

Reply:
oops, I've copy paster it from my react sandbox app. Thank you for noticing it